distributed/worker.py

```diff
@@ -1836,13 +1836,16 @@ def ensure_communicating(self):
 
     def send_task_state_to_scheduler(self, key):
         if key in self.data or self.actors.get(key):
-            try:
-                value = self.data[key]
-            except KeyError:
-                value = self.actors[key]
-            nbytes = self.nbytes[key] or sizeof(value)
```

Comment:
This appeared to just use `getitem` on `nbytes`, which AFAICT is a `dict`. So maybe getting `value` is never needed? If it is, maybe we should be changing this to `.get(...)`.

Reply:
Yeah, good catch.  I don't know.

Reply:
I went ahead and used `.get(...)` instead and raised issue ( https://github.com/dask/distributed/issues/3629 ) to determine whether the fallback to get `value` is needed.